crates/rethnet_eth/src/remote.rs

```diff
@@ -231,24 +231,70 @@ impl RpcClient {
         Ok(success.result)
     }
 
+    fn hash_string(input: &str) -> String {
+        use std::hash::Hasher;
+        let mut hasher = std::collections::hash_map::DefaultHasher::new();
+        hasher.write(input.as_bytes());
+        hasher.finish().to_string()
+    }
+
+    fn make_cache_path(&self, request_body: &str) -> std::path::PathBuf {
+        // TODO: consider using a better path for this directory. currently, for test runs,
```

Comment:
I'd let the end user optionally specify the caching directory. If they don't specify anything use your default. That should be straightforward to explain in the comments of the API